src/models/notion.model.ts

```diff
@@ -128,6 +129,10 @@ const parseNotionProperty = (property: NotionProperty): unknown => {
         (property["title"] as Array<{ plain_text: string }>)[0]?.plain_text ||
         ""
       );
+    case "checkbox":
+      return (
+        (property["checkbox"] as { checkbox: boolean })
```

Comment:
I think here  we want tu return a boolan not an object containing a boolean

`(property["checkbox"] as { checkbox: boolean }).checkbox`